packages/react-sdk/src/useRiverAuthStatus.ts

```diff
@@ -0,0 +1,18 @@
+'use client'
+
+import { AuthStatus } from '@river-build/sdk'
+import { useRiver } from './useRiver'
+
+export const useRiverAuthStatus = () => {
+    const { data: status } = useRiver((s) => s.riverAuthStatus)
+    return {
+        status,
+        isInitializing: status === AuthStatus.Initializing,
+        isEvaluatingCredentials: status === AuthStatus.EvaluatingCredentials,
+        isCredentialed: status === AuthStatus.Credentialed,
+        isConnectingToRiver: status === AuthStatus.ConnectingToRiver,
+        isConnectedToRiver: status === AuthStatus.ConnectedToRiver,
+        isDisconnected: status === AuthStatus.Disconnected,
+        isError: status === AuthStatus.Error,
```

Comment:
What do these booleans really add? Why wouldn’t the end user use the enum?

Reply:
I find it useful when building UI, ill start using it on playground to see if the current DX looks good